rust-runtime/aws-smithy-http/external-types.toml

```diff
@@ -12,20 +11,10 @@ allowed_external_types = [
     "http::response::Builder",
     "http::response::Response",
     "http::uri::Uri",
-    "http::version::Version",
-    "http_body::Body",
-    "http_body::combinators::box_body::BoxBody",
-    "hyper::body::body::Body",
-
-    # TODO(https://github.com/awslabs/smithy-rs/issues/1193): Feature gate Tokio `AsyncRead`
-    "tokio::io::async_read::AsyncRead",
 
     # TODO(https://github.com/awslabs/smithy-rs/issues/1193): Switch to AsyncIterator once standardized
     "futures_core::stream::Stream",
```

Comment:
What is this still needed for?

Reply:
For event-stream. It's worth updating the `TODO` comment to only use `Stream` with the `event-stream` feature.

Reply:
Updated in b8c6029.